code/modules/mob/living/say.dm

```diff
@@ -269,13 +269,15 @@ var/list/headset_modes = list(
 			rendered_message = replacetext(rendered_message, T, "<i style='color: red;'>[T]</i>")
 
 	//AI mentions
-	if(istype(src, /mob/living/silicon/ai) && speech.frequency && speech.job != "AI")
+	if((isAI(src)  || isshell(src)) && speech.frequency && !(findtext(speech.job,"AI")<>0) && (speech.name != src.name))
```

Comment:
how does this compile? the `<>` part
did you test this?

Reply:
yep. findtext reports 0 if it finds nothing, and something else if something was found. it's essentially != 0